src/quo2/components/empty_state/empty_state/view.cljs

```diff
@@ -0,0 +1,48 @@
+(ns quo2.components.empty-state.empty-state.view
+  (:require [quo2.components.buttons.button :as button]
+            [quo2.components.empty-state.empty-state.styles :as styles]
+            [quo2.components.markdown.text :as text]
+            [react-native.core :as rn]
+            [react-native.fast-image :as fast-image]))
+
+(defn empty-state
+  [{:keys        [customization-color image title description blur?]
+    upper-button :upper-button
+    lower-button :lower-button
+    :or          {customization-color :blue}}]
+  [rn/view {:style styles/container}
+   [fast-image/fast-image
```

Comment:
@ulisesmac and I already talked about this, but apparently `FastImage` is meant to be used with remote sources because it's mostly focused on HTTP resources and caching based on cache headers. Does this sound about right to you @flexsurfer? We should use `Image` when the image was loaded via `js/require` and `FastImage` when the image comes from an HTTP server (like our media server).